src/sublime_aio.py

```diff
@@ -58,12 +76,12 @@ def shutdown():
     loop.run_until_complete(loop.shutdown_asyncgens())
     loop.close()
 
-atexit.register(on_exit)
 
 # ---- [ public ] -------------------------------------------------------------
 
-def debounced(delay_in_ms: int) -> Callable:
-    """Schedule coroutine for execution as soon as no events arive for given delay.
+
+def debounced(delay_in_ms: int):
```

Comment:
Isn't the returned `wrapper` a `Callable`?

Reply:
Yes, but I couldn't get the signature right so I relied on pyright's type inference here. Will add a comment about it.